packages/core/src/messagingProtocols/mls/MLSService/MLSService.ts

```diff
@@ -118,7 +118,11 @@ export class MLSService extends TypedEventEmitter<Events> {
 
   public async initClient(userId: QualifiedId, client: RegisteredClient) {
     const qualifiedClientId = constructFullyQualifiedClientId(userId.id, client.id, userId.domain);
-    await this.coreCryptoClient.mlsInit(this.textEncoder.encode(qualifiedClientId), [this.config.defaultCiphersuite]);
+    await this.coreCryptoClient.mlsInit(
+      this.textEncoder.encode(qualifiedClientId),
+      [this.config.defaultCiphersuite],
+      this.config.nbKeyPackages,
```

Comment:
Now consumer can specify how many keys corecrypto should generate initially when registering a new client.